config/questions/en.yml

```
@@ -1,311 +1,307 @@
-Do:
-  you:
-    - recycle
+---
+- Are you:
+  - afraid of going outside at night
+  - religious
+
+- Can you:
+  - cook
+  - dance
+  - draw
+  - play:
+    - any:
+      - sports
+    - baseball
+    - the:
+      - guitar
+      - piano
+      - saxophone
+      - trumpet
+  - ski
+  - speak different languages
+  - swim
+  - yodel
+
+- Do:
+  - you:
     - have:
-      - a:
-        - nickname
+      - a nickname
       - any:
         - siblings
         - pets
     - like:
-      - social networks
       - muffins
-      - video games
-      - the:
-        - city:
-          - or:
-            - country
-          - where you:
-            - live
+      - social networks
+      - the city where you live
```

```suggestion
      - the place where you live
```

Some people live in towns 😉